ch.md

```diff
@@ -0,0 +1,20 @@
+### Ch CLI shell
+
+The ch CLI shell is used for internal testing of Chakra Core but can be used as a standard alone shell with minimal capabilities.
+Be aware that it doesn't provide a stable hosting API so it can change from one release to another.
```

Comment:
I'd say replace this sentence with

`To facilitate testing, a few host capabilities beyond the scope of ECMAScript standard (e.g.  loading files) are also provided in ch.`

Original wording may also confuse someone to believe the shell is hostable.